### PR TITLE
The new feature 2

### PR DESCRIPTION
**Pull request by KindoxPerseo**
This feature integrate with other platforms

### CODE REVIEW

**Review by delmohf — COMMENTED** [review]
I think @KindoxPerseo did a great job with this commit!

File: o1.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ccc1111
 ccc222
+ccc333
```
Comment (delmohf) [inline o1.txt:3]: Great line!

File: o1.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 ccc1111
```
Comment (delmohf) [inline o1.txt:1]: Keep this line always as is.

**Review by delmohf — COMMENTED** [review]
I left you some interesting comments!

File: o1.txt (hunk 1 above)
Comment (KindoxPerseo) [reply o1.txt:3]: Yes it costs to me!

File: o1.txt (hunk 2 above)
Comment (KindoxPerseo) [reply o1.txt:1]: > 
> 
> Keep this line always as is.

Yes Sr.!

File: o1.txt [hunk 3]
```diff
@@ -1,2 +1,3 @@
...
 ccc222
```
Comment (delmohf) [inline o1.txt]: I forgot highlight these two lines!

**Review by delmohf — COMMENTED** [review]
Just an idea about two lines of code

File: o1.txt (hunk 3 above)
Comment (KindoxPerseo) [reply o1.txt]: Yeah! It's a bit late.

**Review by delmohf — APPROVED** [review]

**Comment by KindoxPerseo** [comment]
Trying to reopen

**Pull request merged**